wandb/sdk/internal/system/system_info.py

```diff
@@ -170,6 +171,11 @@ def _save_patches(self) -> None:
 
     def _probe_git(self, data: Dict[str, Any]) -> Dict[str, Any]:
         if self.settings.disable_git:
+            # can't trust relpapth, possibly set before git_disabled
+            data["codePath"] = _get_program_relpath_from_gitrepo(
```

Comment:
We should probably not call `_probe_git` if it's disabled. So let's move this check outside of probe_git?

Reply:
Oh, sorry this is already like this!

Reply:
No I think you were right, we call it every time